dashboard/test/controllers/api/v1/pd/workshop_survey_report_controller_test.rb

```diff
@@ -101,20 +101,29 @@ class Api::V1::Pd::WorkshopSurveyReportControllerTest < ::ActionController::Test
     assert_equal Array.new(3, 'Jaime was very funny'), response_hash['this_workshop']['things_facilitator_did_well']
     assert_equal Array.new(3, 'Jaime was very funny'), response_hash['all_my_local_workshops']['things_facilitator_did_well']
 
-    [organizer, admin].each do |user|
-      sign_in(user)
-      @controller = @controller.class.new
-      get :local_workshop_survey_report, params: {workshop_id: workshop_2.id}
-      assert_response :success
-      response_hash = JSON.parse(@response.body)
-
-      assert_equal 1, response_hash['this_workshop']['how_much_learned']
-      assert_nil response_hash['all_my_local_workshops']['how_much_learned']
-      assert_equal({'Jaime' => 1.0}, response_hash['this_workshop']['how_clearly_presented'])
-      assert_nil response_hash['all_my_local_workshops']['how_clearly_presented']
-      assert_equal({'Jaime' => Array.new(3, 'Jaime was very funny')}, response_hash['this_workshop']['things_facilitator_did_well'])
-      assert_nil response_hash['all_my_local_workshops']['things_facilitator_did_well']
-    end
+    sign_in(organizer)
+    @controller = @controller.class.new
```

Comment:
Ok, well ideally this would be split with common setups. If we can get the describe/it syntax working that would be perfect. Otherwise I think I'd still prefer extracting the setup into a method so we can have 2 test cases. Or convert to an integration test. 

I'm also ok accepting this for now, and cleaning up the test in a subsequent PR